examples/grant-access/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Access Grants demo app.",
   "main": "src/index.js",
   "scripts": {
-    "build": "npm link ../../ && tsc -p tsconfig.json"
+    "build": "tsc -p tsconfig.json",
+    "build:local": "npm link ../../ && tsc -p tsconfig.json",
```

Comment:
Yes, but here I don't use link (or [file linking](https://github.com/inrupt/solid-client-access-grants-js/pull/264#discussion_r902542183)) by default intentionally because it doesn't work with node 14.

Ideally I would resolve that too, but it is too much for one PR (and arguably it is lower priority than my education work).